deploy/014-shortfall-protocolshare.ts

```diff
@@ -0,0 +1,66 @@
+import { ethers } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { getConfig } from "../helpers/deploymentConfig";
+import { getBlockOrTimestampBasedDeploymentInfo, toAddress } from "../helpers/deploymentUtils";
+import { convertToUnit } from "../helpers/utils";
+
+const MIN_POOL_BAD_DEBT = convertToUnit(1000, 18);
+
+const nextBidderBlockOrTimestampLimit = 100; // for block based contracts
+const waitForFirstBidder = 100; // for block based contracts
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+  const { preconfiguredAddresses } = await getConfig(hre.network.name);
```

Comment:
Let's fetch these contracts and deprecate `preconfiguredAddresses`

Reply:
We'll do it in a different PR. VEN-2596